falcon_telegraf/__init__.py

```diff
@@ -1,9 +1,7 @@
 from .log_hits import LogHits
-from .log_hits import LogHitsContextAware
 from .timer import Timer
 
 __all__ = [
     'LogHits',
-    'LogHitsContextAware',
     'Timer',
```

Comment:
Maybe instead mark `LogHitsContextAware` as deprecated and (log.warn in init) and apart from that remove the body of that class. This way you provide a pain-free method of upgrade with this highly beneficial change.